contracts/base/EtherCollateralLoans.sol

```diff
@@ -21,16 +21,18 @@ pragma experimental ABIEncoderV2;
 import "./LoansBase.sol";
 
 // Interfaces
-import "../interfaces/EtherLoansInterface.sol";
+import "../interfaces/LoansInterface.sol";
 
 
-contract EtherLoans is EtherLoansInterface, LoansBase {
+contract EtherCollateralLoans is LoansInterface, LoansBase {
```

Comment:
As I mentioned in TokenCollateralLoans, I think the LoansInterface can be moved to LoansBase contract, and then move all the _emitXXXXEvent implementations to LoansBase. So, parent contracts (and LoansBase) can call them.